schedule/yam/agama_puppetee.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
agama_auto.yaml already exists, please use that file.

Reply:
ok, thanks